__tests__/components/ui/SideNavigation/_partials/Action.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { ActivableLink } from "@/components/ui/SideNavigation/_partials/Action";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("Activable link", () => {
+  const item = {
+    subitems: [],
+    link: "/example",
+  };
+  it("should render the link with subitems", () => {
+    render(
+      <ActivableLink
+        item={{
+          subitems: [{ label: "subitem", link: "subitem link" }],
+          link: "/dummy",
+        }}
+      >
+        Link test
+      </ActivableLink>
+    );
```

Comment:
Can you create a rendern function for this component, you can refer to the button tests